dev-libs/tomsfastmath/tomsfastmath-0.13.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="TomsFastMath is a fast public domain large integer arithmetic library"
+HOMEPAGE="https://www.libtom.net/TomsFastMath/ https://github.com/libtom/tomsfastmath"
+SRC_URI="https://github.com/libtom/tomsfastmath/releases/download/v${PV}/tfm-${PV}.tar.xz"
+LICENSE="Unlicense"
+
+SLOT="0/1"
+KEYWORDS="~amd64"
+
+BDEPEND="virtual/pkgconfig"
```

Comment:
pkg-config isn't used by the build system for tomsfastmath itself, unless I'm grepping wrong

Reply:
Oops. Thank you